sdv/constraints/tabular.py

```diff
@@ -941,20 +908,50 @@ def _transform(self, table_data):
                 Transformed data.
         """
         low = table_data[self.low_column_name]
+        middle = table_data[self.middle_column_name]
         high = table_data[self.high_column_name]
 
-        data = logit(table_data[self.middle_column_name], low, high)
-        table_data[self._transformed_column] = data
-        table_data = table_data.drop(self.middle_column_name, axis=1)
+        if self._is_datetime:
+            diff_column_1 = get_datetime_diff(middle, low, self._dtype)
+            diff_column_2 = get_datetime_diff(high, middle, self._dtype)
+        else:
+            diff_column_1 = middle - low
+            diff_column_2 = high - middle
 
-        return table_data
+        self.low_diff_column_name = modify_existing_name(
+            self.low_diff_column_name, table_data.columns
+        )
+        self.high_diff_column_name = modify_existing_name(
+            self.high_diff_column_name, table_data.columns
+        )
```

Comment:
I think we can do this during the fit and create the variables there

Reply:
Yes thanks, done in [3a3cc68](https://github.com/sdv-dev/SDV/pull/1404/commits/3a3cc68a54677e2cca58f8417712fe9862497998).